packages/Confirmation/src/Confirmation.js

```diff
@@ -51,7 +51,8 @@ const Confirmation = props => {
   } = props;
   const confirmButtonRef = React.useRef(null);
   const triggerRef = React.useRef(null);
-  const confirmId = React.useRef(uuid());
+  const confirmId = React.useRef(uuid()).current;
+  let popoverKey = uuid();
```

Comment:
I think this one can be ref as well. In the useEffect, we can do `popoverKeyRef.current = uuid()`